.github/workflows/ci.yml

```diff
@@ -40,13 +40,13 @@ jobs:
           sudo cp binaryen-$BINARYEN_VERSION/bin/wasm-opt /usr/bin/
 
       - name: Check Code Formatting
-        run: cargo fmt --all --check
+        run: __GEAR_WASM_BUILDER_NO_FEATURES_TRACKING=1 cargo fmt --all --check
```

Comment:
what's that for? __GEAR_WASM_BUILDER_NO_FEATURES_TRACKING

Reply:
When unset this wasm builder forces build script to get re-run on every next build run. Done for the sake of tracking runs with different feature set and propagate it down to wasm building process